fhir-term-graph/pom.xml

```diff
@@ -101,5 +101,9 @@
             <groupId>commons-cli</groupId>
             <artifactId>commons-cli</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.ibm.cos</groupId>
+            <artifactId>ibm-cos-java-sdk</artifactId>
+        </dependency>
```

Comment:
We were already using this in fhir-bulkdata-webapp, but this is a new dependency for the fhir-server-webapp. I think it pulls in much of the aws sdk.
Since its not actually needed by the server during runtime, I think we should work to split out this new loading tool work so that it doesn't end up in the webapp.

Reply:
agreed https://github.com/IBM/FHIR/issues/2799

Reply:
ah, I read that as more of creating a utility class to encapsulate the code that accesses COS.  i didn't think it went quite as far as what i'm suggesting here which is to split fhir-term-graph into two totally separate modules:
1. the loading stuff the creates/populates the graph db
2. the provider stuff that uses the graph and ships with the server